nss-cohort-37/Bangazon-Site-Red-Jaguars
# completed stuff

# Description

included images in product index view. also made max image size on product detail view.

Fixes #60 

## Type of change


# Testing Instructions

git checkout jv-imageIndex
make sure images show up in index view
make sure images have a max size of 75px and 500px in details view
make sure you can submit a null image

# Checklist:
